desc/plotting.py

```diff
@@ -1724,6 +1726,156 @@ def plot_surfaces(eq, rho=8, theta=8, phi=None, ax=None, return_data=False, **kw
     return fig, ax
 
 
+def poincare_plot(
```

Comment:
should we add a warning if there is a coil going through the plasma? Because I think there's no reason to field line trace if that's happening and it's taken me a while to figure out before (but maybe a separate issue)